app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -69,6 +68,8 @@ angular.module('dashboard')
                     var topic = self.meeting.topicList[j];
                     if (angular.isObject(topic)) {
                         topic.isModified = (changedTopicGuidArray.indexOf(topic.topicGuid) > CONST.NOTFOUND);
+                        topic.userPersonGuid = self.meeting.userPersonGuid;
```

Comment:
Is this the right place to copy properties from meeting to agenda item (topic)? Should it be done in `meetingStatusCtrl initialisation`?